src/program/processor/new_order.rs

```diff
@@ -62,7 +108,11 @@ impl MultipleOrderPacket {
             bids,
             asks,
             client_order_id,
-            reject_post_only,
+            failed_multiple_limit_order_behavior: if reject_post_only {
+                FailedMultipleLimitOrderBehavior::RejectPostOnly
+            } else {
+                FailedMultipleLimitOrderBehavior::AmendOnCross
+            },
```

Comment:
should we change the method signature to include fail_silently_if_insufficient_funds?